setup.cfg

```diff
@@ -3,3 +3,19 @@ ignore = E251,E252,E501,W503
 
 [flake8]
 ignore = E402,W503,E501,E203
+
+[tool:pytest]
```

Comment:
On the website of pytest they mention:

> Warning
Usage of setup.cfg is not recommended unless for very simple use cases. .cfg files use a different parser than pytest.ini and tox.ini which might cause hard to track down problems. When possible, it is recommended to use the latter files, or pyproject.toml, to hold your pytest configuration.

I do not know whether you saw this or how important it is, but it could be good to keep in mind if there ever are any problems.

Reply:
Good to know... I think this should fall under "simple use case" (it seems to work at least) but we can bear this in mind going forwards. Ideally we would just use `pyproject.toml` but flake8 does not currently support it